src/vs/workbench/contrib/debug/browser/debug.contribution.ts

```diff
@@ -532,6 +532,16 @@ configurationRegistry.registerConfiguration({
 			default: 'allEditorsInActiveGroup',
 			scope: ConfigurationScope.LANGUAGE_OVERRIDABLE
 		},
+		'debug.saveBeforeTest': {
```

Comment:
This should be in the test settings, not the debug settings

Reply:
Thanks for the reminder

Reply:
Thanks for the reminder!

Reply:
I tweaked it a bit and used the bool type，which seems to be sufficient for now